CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Make `assert_covers` recursive (gh-379).
```

Comment:
Usually, we add the new entries to the end of the list.

Reply:
My bad, fixed, thanks!